src/main/java/org/spongepowered/api/item/inventory/ItemStack.java

```diff
@@ -130,4 +132,22 @@
      * @return The level of the enchantment, or -1 if the enchantment is not applied to this stack
      */
     int getEnchantment(Enchantment enchant);
+    
```

Comment:
`AttributeHolder` is the target where attributes apply to.
`ItemStack` is more a source of attribute modifiers. Maybe a new interface `AttributeSource` is a good addition, this could be used in the events as well.